### PR TITLE
feat(navbar): ENG-268 navbar redesign

### PR DESCRIPTION
**Pull request by danielgek**
## 🗃 Story Description

_Link to Shortcut card, if PR is part of a story._

## 🧪 Test Suite
- Navbar Actions
  - [ ] Switch between light and dark mode across pages
  - [ ] Network Switcher
  - [ ] Wrong Network Modal
- Homepage Markets Navigation
  - [ ] Filters
  - [ ] Sorting
  - [ ] Search
- Pages Content
  - [ ] Homepage
  - [ ] Create Market Page
  - [ ] Market Page
  - [ ] Portfolio
- Trading (**Optional** - only if PR involves related components or `polkamarkets-js`)
  - [ ] Slider + MAX button
  - [ ] Buy Outcome Shares
  - [ ] Sell Outcome Shares
  - [ ] Add Liquidity 
  - [ ] Remove Liquidity
 
  ## 📝 Footnotes
  _Noting to add._

### CODE REVIEW

**Comment by danielgek** [comment]
This PR also 

Closes ENG-274 
Closes ENG-275

File: src/components/Header/ProfileMenu/ProfileMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+              <>
+                <div className={styles.menuProfileAvatar}>
+                  <Avatar
+                    $radius="lg"
+                    className={styles.menuAvatar}
+                    src={socialLoginInfo?.profileImage}
+                    alt={username || 'avatar'}
+                  />
+                  <div className={styles.menuProfileInfo}>
+                    <span className={styles.menuUserName}>Unamed</span>
```
Comment (rsmarques) [inline src/components/Header/ProfileMenu/ProfileMenu.tsx:135]: Typo: `Unnamed`

Shouldn't it be `{username || 'Unnamed'}` @danielgek? It's already computed above

**Review by rsmarques — CHANGES_REQUESTED** [review]
Looking good @danielgek! 

A few notes:

- Check my `username || Unnamed` comment above
- Dropdown colors aren't showing up properly on my side (probably due to my white mode system config)

image

**Comment by danielgek** [comment]
@rsmarques fixed both issues and rebased

**Review by rsmarques — APPROVED** [review]
@danielgek looks good! Just missing a rebase/merge from `feat/v3` so we can merge it 🙂

**Pull request merged**